test/helpers/steps/teardown/operator.go

```diff
@@ -7,15 +7,13 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/csi"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/oneagent"
 	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/operator"
-	"github.com/Dynatrace/dynatrace-operator/test/helpers/kubeobjects/namespace"
 	"sigs.k8s.io/e2e-framework/pkg/features"
 )
 
 func UninstallOperatorFromSource(builder *features.FeatureBuilder, testDynakube dynatracev1beta1.DynaKube) {
 	addCsiCleanUp(builder, testDynakube)
 	addNodeCleanUp(builder, testDynakube)
 	builder.WithTeardown("operator manifests uninstalled", operator.UninstallViaMake(testDynakube.NeedsCSIDriver()))
-	builder.WithTeardown("deleted operator namespace", namespace.Delete(testDynakube.Namespace))
```

Comment:
because the e2e pipeline has a special secret to access the internal registry 
- this secret is created during pipeline creation
- the dev tenant's endpoint we use gives results for the internal registry
- if we delete the namespace between test this secret is removed, so this test wouldn't be able to run